### PR TITLE
Jordan branch

### PR DESCRIPTION
**Pull request by HeirJordan3**
Added the HTML version of my README.

### CODE REVIEW

File: lesson_01/Jordan/ReadMe.html
Comment (anthonydmays) [inline lesson_01/Jordan/ReadMe.html:1]: There are some significant differences between the visual appearance of this file and the markdown version. Please fix.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_01/Jordan/ReadMe.html
Comment (anthonydmays) [reply lesson_01/Jordan/ReadMe.html:1]: Notably, the kobe picture is missing and the section about family and pets. There are other things too, so keep working on it.

Comment (HeirJordan3) [reply lesson_01/Jordan/ReadMe.html:1]: Made the changes required should be good to go!

File: lesson_01/Jordan/ReadMe.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+    <h2>💬 Topics I Enjoy</h2>
+    <ul>
+        <li><strong>Sports Talk:</strong> Game night highlights or debating the GOAT? I'm in.</li>
+        <li><strong>Snowboarding Stories:</strong> Slopes, snow, and everything in between.</li>
+        <li><strong>Family Feats:</strong> The core of my life, I cherish conversations about family.</li>
+        <li><strong>Photo Ops:</strong> Peep my photography skills on <a href="https://www.instagram.com/jordanxeldridge/">Instagram</a>.</li>
+    </ul>
+
+    <h3>🍳 Chef Jordan in the House</h3>
```
Comment (anthonydmays) [inline lesson_01/Jordan/ReadMe.html:65]: Not matching the consistency of the style in your markdown version

File: lesson_01/Jordan/ReadMe.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    <p>First off, let's play a game to break the ice. I'll throw out three facts about myself, but catch this - one of them is a fib. Your mission, should you choose to accept it, is to sniff out the lie. Ready? Here we go:</p>
+    <ol>
+        <li>Born in the chill vibes of Delaware.</li>
+        <li>My family gatherings are a mini festival with 8 siblings.</li>
+        <li>Got to be in the arms of the basketball legend, Kobe Bryant.</li>
+    </ol>
+    <p><img src="https://media4.giphy.com/media/DOEXrsUpJ6IIP0GfM9/200.gif?cid=6104955enoqzvst5iqhm3h1wpje5919uesax36r0lyv5c6wi&ep=v1_gifs_translate&rid=200.gif&ct=g" alt="GIF of excitement"></p>
+    <p>Okay, drumroll please 🥁... So, the whole Delaware thing? It's only half the story. Born in Ohio, then did my growing-up thing in Darby Township, but high school? That was all Delaware, starting 2010. And yeah, Kobe Bryant holding me is a real deal - pretty wild, right? Plus, toss in 8 siblings to the mix, and you've got my kinda crazy, totally awesome life!</p>
+    <p><img src="https://github.com/HeirJordan3/CodeDifferently_README/blob/main/assets/Kobe.JPG?raw=true" alt="" width=30%></p>
```
Comment (anthonydmays) [inline lesson_01/Jordan/ReadMe.html:24]: This appears differently sized in your markdown version.

File: lesson_01/Jordan/ReadMe.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        <li><strong>Text Me:</strong> If a prompt response is critical and I haven't answered my phone, please send a text for a quicker reply.</li>
+    </ul>
+
+    <h2>🗣 Feedback Preferences</h2>
+    <p>Feedback is a gift, and I value it greatly as it contributes to personal and professional growth.</p>
+    <ul>
+        <li><strong>One-on-One Setting:</strong> For feedback, especially personal or sensitive in nature, I prefer face-to-face conversations. It ensures clarity and allows for a more meaningful exchange.</li>
+    </ul>
+
+    <h2>🎯 Goals for the Coming Year</h2>
```
Comment (anthonydmays) [inline lesson_01/Jordan/ReadMe.html:51]: Missing some content in this section.

**Review by anthonydmays — CHANGES_REQUESTED** [review]
Still noting some significant differences, but closer. Please adjust.

**Review by HeirJordan3 — COMMENTED** [review]
made some changes

**Review by anthonydmays — APPROVED** [review]

**Pull request merged**